back/prisma/scripts/new-work-site-model.ts

```diff
@@ -0,0 +1,36 @@
+import { prisma } from "../../src/generated/prisma-client";
+import { Updater, registerUpdater } from ".";
+
+@registerUpdater(
+  "Set work site",
+  `Migrate old "pickupSite" field to new "workSite*" fields`
+)
+export class SetWorkSiteUpdater implements Updater {
+  run() {
+    console.info("Starting script to set work site...");
+
+    try {
+      // We migrate `emitterPickupSite` to `emitterWorkSiteInfos`
+      // as we have no way of structuring the existing `emitterPickupSite` data...
```

Comment:
J'ai regardé en prod il y a 110 enregistrements dont le `emitterPickupSite` n'est pas "" ou "-", à voir si ça ne vaut pas le coup de les migrer à la main

Reply:
110 à la main c'est beaucoup ! 😱  mais je vais voir si on peu pas automatiser un paquet avec l'api `adresse.data.gouv.fr`. A premiere vue c'est jouable pour au moins 50% des résultats

Reply:
Ca marche pas mal avec l'api. Ce qui reste est souvent vraiment "in-structurable". On pourra faire du ménage à la main pour les derniers dont le résultat de l'API ne parait pas satisfaisant